pkgs/servers/consul/default.nix

```diff
@@ -2,7 +2,7 @@
 
 buildGoModule rec {
   pname = "consul";
-  version = "1.8.0";
```

Comment:
Shouldn't the commit message a be `1.8.0 -> 1.8.6` then, making it technically not a cherry-pick but a security upgrade?

Not trying to be pedantic, I was just thinking that it might be odd to have `consul: 1.8.5 -> 1.8.6` in the 20.09 git log when the upgrade is larger than that.

In any case, I approve of the upgrade to the security-fixed version.

Reply:
it should, i forgot about it when editing the commit

Reply:
done